### PR TITLE
1.0.3 Sections 2.1-2.3 in Data

### PR DESCRIPTION
**Pull request by andyjohnson**
This has a little wordsmithing and some vocabulary updating, some white space fixing, and capitalization fixes.

### CODE REVIEW

File: xAPI-Data.md [hunk 1]
```diff
@@ -132,10 +126,9 @@ just the sum of its parts.
 
 ###### Details
 
-All the properties used in Statements are restricted to certain types, and those types
-constrain the behavior of systems processing Statements. For clarity, certain key
-requirements are documented here, emphasizing where compliant systems have a responsibility
-to act in certain ways.
+All of the properties used in Statements are restricted to certain data types. For clarity, key requirements are 
+documented here, emphasizing where Clients (especially Learning Record Stores (LRSs)) have a responsibility to act
```
Comment (garemoko) [inline xAPI-Data.md]: Clients is everybody except LRSs, right?

**Comment by garemoko** [comment]
Aside from the comment about use of client here +1. "conformant systems" is probably ok?

**Comment by andyjohnson** [comment]
This was the one change I was gun-shy on.  I'm still trying to avoid "systems".  How about "those conformant with the spec"?

**Comment by garemoko** [comment]
@andyjohnson didnt we agree on a term somewhere for all technologies that adopt? I cant remember whether we settled on system or component.

**Comment by andyjohnson** [comment]
I'm okay with component, but I wonder how possible it is to avoid naming
them.

On Thu, Mar 3, 2016 at 5:47 AM, Andrew Downes <notifications@github.com>
wrote:

> @andyjohnson <https://github.com/andyjohnson> didnt we agree on a term
> somewhere for all technologies that adopt? I cant remember whether we
> settled on system or component.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adlnet/xAPI-Spec/pull/889#issuecomment-191703335>.
>



-- 
Andy Johnson
ADL Technical Team
608-318-0049

**Comment by andyjohnson** [comment]
Went with component.

**Comment by garemoko** [comment]
+1

**Comment by ljwolford** [comment]
+1

**Pull request merged**